src/include/storage/store/rel_table_data.h

```diff
@@ -145,6 +146,7 @@ class RelTableData final : public TableData {
     bool delete_(transaction::Transaction* transaction, common::ValueVector* srcNodeIDVector,
         common::ValueVector* relIDVector) const;
 
+    // TODO(Guodong): This function is not used. Should be removed.
```

Comment:
Will clean this up.

Reply:
OK, will keep these that I spot here: https://github.com/kuzudb/kuzu/issues/3729.